modules/assim.sequential/inst/NEFI/US_Syv/syv.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0"?>
+<pecan>
+ <info>
+  <notes></notes>
+  <userid>-1</userid>
+  <username></username>
+  <date>2020/03/10 13:18:24 +0000</date>
+ </info>
+ <outdir>/fs/data3/kzarada/NEFI/US_Syv/output/PEcAn_1000012661/</outdir>
+ <database>
+  <bety>
+   <user>bety</user>
+   <password>bety</password>
+   <host>psql-pecan.bu.edu</host>
+   <dbname>bety</dbname>
+   <driver>PostgreSQL</driver>
+   <write>false</write>
+  </bety>
+  <dbfiles>/fs/data3/kzarada/pecan.data/dbfiles/</dbfiles>
+ </database>
+ <pfts>
+  <pft>
+   <name>temperate.coniferous</name>
+   <constants>
+    <num>1</num>
+   </constants>
+  </pft>
+ </pfts>
+ <meta.analysis>
+  <iter>3000</iter>
+  <random.effects>FALSE</random.effects>
+ </meta.analysis>
+ <ensemble>
+  <size>100</size>
+  <variable>NEE</variable>
+  <start.year>2020</start.year>
+  <end.year>2020</end.year>
+  <samplingspace>
+   <parameters>
+    <method>uniform</method>
+   </parameters>
+   <met>
+    <method>sampling</method>
+   </met>
+   <soil>
+    <parent>parameters</parent>
+   </soil>
+   <vegetation>
+    <parent>soil</parent>
+   </vegetation>
+  </samplingspace>
+ </ensemble>
+ <model>
+  <id>1000000030</id>
+  <default.param>/fs/data3/kzarada/US_WCr/data/WillowCreek.param</default.param>
+ </model>
+ <workflow>
+  <id>1000012661</id>
+ </workflow>
+ <run>
+  <site>
+   <id>622</id>
+   <met.start>2020-02-25</met.start>
+   <met.end>2020-03-24</met.end>
+  </site>
+  <inputs>
+   <met>
+    <source>NOAA_GEFS_downscale</source>
+    <output>SIPNET</output>
+   </met>
+  </inputs>
+  <start.date>2020-03-08</start.date>
+  <end.date>2020-03-24</end.date>
```

Comment:
Should start/end dates be hard-coded in the template? Seems like your code should be filling in data-specific info on the fly.

Reply:
The start and end dates are changed using the generate.gefs.xml.R file. This also updates the workflow ID and directories

Reply:
I just added in a README file that goes through which files are needed for the forecasts and how to add a new forecast to the system